dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/MetadataDumper.java

```diff
@@ -130,6 +134,32 @@ public void run(@Nonnull ConnectorArguments arguments) throws Exception {
   private <T> T runTask(
       @Nonnull TaskRunContext context, @Nonnull TaskSetState.Impl state, Task<T> task)
       throws MetadataDumperUsageException {
+    T t = executeTask(context, state, task);
+
+    numberOfCompletedTasks += 1;
+
+    long averageTimePerTaskInMillisecond =
+        (System.currentTimeMillis() - startTimeInMilliseconds) / numberOfCompletedTasks;
+
+    int percentFinished = numberOfCompletedTasks * 100 / totalNumberOfTasks;
+    String progressMessage = percentFinished + "% Completed";
+    PROGRESS_LOG.info(progressMessage);
+    LOG.info(progressMessage);
```

Comment:
I've suggested using the `additivity` attribute in order to exclude the progress logs from the console. With the new requirement to log both to the file and to console, this no longer makes sense. I would suggest to leave just the logging to `PROGRESS_LOG` and remove the `additivity` attribute, which will automatically cause logback to log to both.